### ISSUE TITLE
USWDS - Bug: Selection of a link in mobile menu is not smooth

### ISSUE THREAD

**Issue by DK1999** [uswds/uswds#5058]
### Describe the bug

When a user chooses a page in the mobile menu, the expectation is for the next view that appears to the user to be the new, chosen page. Instead, the menu closes. The current (old) page appears for a time. Then, the new, chosen page loads. The result is flashy and clunky. On a website like USA.gov, where each page is so similar in appearance, it can also be confusing.
The current www.usa.gov has the expected behavior. Check it out! (The screenshot .gif attached below is for a test version of the next usa.gov on federalist. It uses the mobile menu closing behavior of USWDS).
You can also see the flashy mobile menu closing experience on the USWDS website, but I didn't screenshot a gif of that.
Our mobile menu gets used a lot, and this behavior is not a smooth experience.

### Steps to reproduce the bug

Use a mobile phone or mobile replicator in dev tools.
Go to USWDS website like https://designsystem.digital.gov/design-principles/.
Open Mobile Menu.
Choose a new page.
Observe that the mobile menu closes, revealing old page. Then the new page loads.

### Expected Behavior

Mobile menu stays open when new page is clicked. Then new page loads.

### Screenshots

![2022-11-30_13-39-02 (1)](https://user-images.githubusercontent.com/25777335/204887511-4047b49a-a575-49d9-a211-92f383e6f7f2.gif)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by mahoneycm** [comment]
Hello @DK1999, thank you for submitting this bug!

I am able to reproduce, however, this behavior appears to be intended as some navigations links may link to in-page content and  not load a new page. 

Consider component submenu links on USWDS
https://user-images.githubusercontent.com/25211650/232828281-f98b1b33-4b69-4891-9750-89423cd4d8c2.mov

I do think this has potential to be a feature enhancement! Specify linking to a new page, which will leave the menu open and wait for the page load to take the user away from the menu, while in-page navigation links trigger the menu to close.

I'll discuss with the team and see if we can dive deeper!

Thanks again,
Charlie

**Comment by amyleadem** [comment]
## Next steps
Add unit tests to PR #5513 to automate testing

**Comment by mejiaj** [comment]
A new bug was discovered. 

We'll create an issue for it to resolve, so that it can unblock the attached PR.

**Comment by dct2023** [comment]
After connecting with @mejiaj we are moving this to the backlog given this work is blocked and not a priority for upcoming deliverables.

**Comment by annepetersen** [comment]
Hi there: this is one of many submitted PRs/issues that have languished here. USWDS is currently understaffed, so I'm closing items that aren’t as high on the priority list right now, including this one, unfortunately.
